### PR TITLE
changed user photo by default in user list and user icon

### PR DESCRIPTION
**Pull request by tatyana-budnyk**

### CODE REVIEW

File: src/pages/Profile/Profile.js [hunk 1]
```diff
@@ -52,10 +52,14 @@ class Profile extends Component {
   render () {
     return (
       <div className="profile-container">
-      <img
-          src={this.props.user.avatar}
-          className="user-image-profile"
-      />
+      <div className={this.props.user.avatar ?
+        'user-image' + 'display-user' : 'user-image'}>
```
Comment (totoshk) [inline src/pages/Profile/Profile.js]: 'user-image' + 'display-user' = user-imagedisplay-user

File: src/pages/Profile/Profile.js [hunk 2]
```diff
@@ -52,10 +52,14 @@ class Profile extends Component {
...
+        <img className="user-image-profile" src={this.props.user.avatar} />
+      </div>
+      <div className={this.props.user.avatar ?
+        'user-image' : 'user-image' + 'display-user'}>
```
Comment (totoshk) [inline src/pages/Profile/Profile.js]: The same mistake.

File: src/pages/Profile/Profile.js (hunk 1 above)
Comment (tatyana-budnyk) [reply src/pages/Profile/Profile.js]: fixed

**Review by totoshk — APPROVED** [review]
Approve

**Review by LanaAvry — APPROVED** [review]

**Pull request merged**